bitnami/keycloak/21/debian-11/prebuildfs/opt/bitnami/scripts/libvalidations.sh

```diff
@@ -207,6 +229,33 @@ validate_ipv4() {
     return $stat
 }
 
+########################
+# Validate if the provided argument is a valid IPv6 address
+# Arguments:
+#   $1 - IP to validate
+# Returns:
+#   Boolean
+#########################
+validate_ipv6() {
+    local ip="${1:?ip is missing}"
+    local stat=1
+
+    if [[ $ip =~ ^[0-9a-fA-F]{1,4}(:[0-9a-fA-F]{1,4}){7}$ ]]; then
+        read -r -a ip_array <<< "$(IFS=':' read -ra arr <<< "$ip" && echo "${arr[@]}")"
```

Comment:
Can you also replace this line with the one below to be consistent with the `validate_ipv4` method?

```
read -r -a ip_array <<< "$(tr ':' ' ' <<< "$ip")"
```

Reply:
the new function doesn't use read anymore